lib/vagrant-openshift/constants.rb

```diff
@@ -30,7 +34,21 @@ def self.openshift_repos
         }
       end
 
-      def self.git_branch_current
+       def self.aggregated_logging_repos
+        {
+          'origin-aggregated-logging' => 'https://github.com/openshift/origin-aggregated-logging.git'
+        }
+      end
+
+      def self.get_repos(reponame)
```

Comment:
Changed to `repos_for_name` - i.e. return the repos associated with the given repo name